src/popup/pages/SignMessage.vue

```diff
@@ -20,7 +20,7 @@ export default defineComponent({
           app: {
             name: callbackOrigin.value?.host,
             host: callbackOrigin.value?.host,
-            url: callbackOrigin.value?.origin,
+            url: callbackOrigin.value,
```

Comment:
Is this correct? Here the instance of `URL` (class) or null is passed. Maybe `toString` method should be applied here?